reference/entry.json

```diff
@@ -2754,6 +2814,18 @@
         },
         "description": "The language of the request body properties"
       },
+      "labelName": {
+        "name": "labelName",
+        "in": "path",
+        "required": true,
+        "schema": {
+          "type": "string",
+          "example": "MyLabel",
+          "minLength": 1,
+          "maxLength": 255
+        },
+        "description": "The label to add to an event, place or organizer. The label name should be longer than 1 character but shorter than 255 characters. The label name should not contain semicolons."
```

Comment:
Looking at the documentation for adding a label to an event, it seems like labels have more restrictions: https://documentatie.uitdatabank.be/content/json-ld/latest/events/event-label.html

> Labels should only start with a letter or number
> Labels should have a minimum length of 2 characters
> Labels should have a maximum length of 50 characters
> Labels should only contain letters, numbers, dash - or underscores _

We should put the first and last restrictions in a regex and add it here. And use the same regex in the code (maybe it already exists?)

Reply:
It also says that label should be 50 characters max while the description here says 255?

Reply:
I took the rules from the `LabelName` value object. I don't think the guidelines were ever applied.

Reply:
Okay then I would suggest:

- Add a regex just for the semicolons for now (and use that in the code and in the error response)
- Check with Corneel if we need to make it stricter in reality (but can be in a follow-up ticket / PR and we need to be careful if this won't break more than it would do good)

Reply:
> and use that in the code and in the error response

I guess using the regex as a hardcoded string and not through the JSON schema files because this is part of the URL instead of the request body?

Reply:
I maybe think it's better to just provide a decent error message here https://github.com/cultuurnet/udb3-silex/pull/792/files#diff-fc2ce84b6415b068b0757e3afae487e75c58b53deaa6c85a1e9b8e57fd499edaR107